src/backend/distributed/commands/statistics.c

```diff
@@ -631,6 +587,49 @@ GetExplicitStatisticsIdList(Oid relationId)
 }
 
 
+/*
+ * GetAlterIndexStatisticsCommands returns the list of ALTER INDEX .. ALTER COLUMN ..
+ * SET STATISTICS commands, based on non default targets of the index with given id.
```

Comment:
probably we don't need to change this function's place ? (to keep change set minimum)

Reply:
I did it to keep non-static functions on top, and static functions at the bottom of the file

Reply:
keeping public functions at the top is not really a policy we have (though it could be nice)